tests/lowering/misc/test_if.py

```diff
@@ -19,6 +19,7 @@ def forward(self, x):
     "input_shape",
     [(4, 4)],
 )
+@pytest.mark.skip(reason="Temporarily disabled due to reset issue in ttnn.sum")
```

Comment:
If there is an issue to track this issue, the issue number can be added here so we won't turn them back after the problem is resolved

Reply:
agreed

Reply:
I fire this issue https://github.com/tenstorrent/pytorch2.0_ttnn/issues/499 to track